e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -573,6 +573,42 @@ test.describe('Example Imagery in Tabs view', () => {
     test.fixme('If the imagery view is not in pause mode, it should be updated when new images come in');
 });
 
+test.describe('Example Imagery in Time Strip', () => {
+    test('ensure that clicking a thumbnail loads the image in large view', async ({ page, browserName }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/5632'
+        });
+        await page.goto('./', { waitUntil: 'networkidle' });
+        const timeStripObject = await createDomainObjectWithDefaults(page, {
+            type: 'Time Strip',
+            name: 'Time Strip'.concat(' ', uuid())
```

Comment:
Nice! I wonder if this should be the default object naming behavior for `createDomainObjectWithDefaults` ?

Reply:
I'm not sure, but it could be desirable for completely isolated tests. Though it could possibly undermine the goal of domain object reusability if we ever initially seeded a batch of domain objects for tests.

Reply:
Good point.

Reply:
We could add a `ensureUnique` flag that would append a UUID to whatever name was provided?

Reply:
That's a good idea! But I think let's hold off for now on modifying the appAction since it's 'proving' itself in the wild right now. If we end up using the `uuid()` naming pattern in more places, then we can move it in later